utils/validate-facility-ids/index.ts

```diff
@@ -0,0 +1,32 @@
+import { difference } from 'lodash';
+import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
+import { mongoDbClient } from './database-client';
+import { getUniqueFeeRecordFacilityIds, getUniqueTfmFacilitiesUkefFacilityIds } from './helpers';
+
+const run = async (): Promise<void> => {
+  const dataSource = await SqlDbDataSource.initialize();
+  try {
+    const tfmFacilitiesUkefFacilityIds = await getUniqueTfmFacilitiesUkefFacilityIds();
+    const feeRecordFacilityIds = await getUniqueFeeRecordFacilityIds(dataSource);
+
+    const feeRecordFacilityIdsNotInTfmFacilities = difference(feeRecordFacilityIds, tfmFacilitiesUkefFacilityIds);
+
+    if (feeRecordFacilityIdsNotInTfmFacilities.length === 0) {
+      console.info('All fee record facility ids exist in the tfm facilities collection');
+      return;
+    }
+
+    feeRecordFacilityIdsNotInTfmFacilities.forEach((facilityId) => {
+      console.info(`Facility id '${facilityId}' exists in the FeeRecord table but not in the "tfm-facilities" collection`);
+    });
```

Comment:
Should we return `feeRecordFacilityIdsNotInTfmFacilities`? Would also avoid having a `Promise<void>` return type

Reply:
This util is only here to display which facility ids do not match up - there is not benefit in returning the value (it would be immediately cleared up by the GC due to the script ending) and I don't see issues with returning `Promise<void>`